src/commands/utils/clusters.ts

```diff
@@ -16,6 +16,13 @@ export interface ClusterARMResponse {
     readonly type: string;
 }
 
+export enum ClusterState {
```

Comment:
If this is an overall 'cluster' state, we could end up with these states increasing exponentially as we combine more features. For example, say we add in scaling and upgrade we might end up with things like 'StartedAndScalingUp', along with a complicated translation layer between RP-level states (provisioning state, power state, etc.) and our VSCode representation of cluster states.

However, I see their utility here. I think my worry above could be sidestepped completely if we just rename this enum to `ClusterStartStopState`. What do you reckon?